### DISCUSSION TITLE
Hello, I'd love to help!

### DISCUSSION THREAD

**Discussion by JManikiza** [govuk-react/govuk-react#1207] [General]
Hello govuk-react team!

Through university I was in a module to create a full-stack web application in a group. The module consisted of work-based learning where NetCompany had set us a task to design a GP services system using your govuk-react. If you desire you could check my repo: https://github.com/JManikiza/SoftwareDevelopment-Group-E-Project

I was in charge of designing the patient profile, profile changes and GP services and connecting it to the databases via PHP. I'd love to help more documentation for certain components and how they would link better in the code. 

Now what I really wanted to enquire about is it possible to join the volunteer team or can I only contribute through posting discussions? 

I've been working with this library for the past month and a half and feel I could offer a lot.

Kind regards,

Joven Manikiza

**Comment by penx** [comment]
Hi @JManikiza , thanks for getting in touch! I need to figure out a way to involve others in the future of this library, and set out a road map that volunteers can work to.

See the discussion at #1200, I'm going to close this discussion in favour of that one so it can all be in one place.
